tests/integration/components/navbar-test.js

```diff
@@ -19,13 +19,6 @@ module('Integration | Component | navbar', function (hooks) {
       this.isLoggedIn = false;
     });
 
-    function generateAuthURL() {
-      const originURL = window.location.href;
-      if (!originURL) return AUTH.SIGN_IN;
-      const signInLink = AUTH.SIGN_IN + '&state=' + originURL;
-      return signInLink;
-    }
```

Comment:
Removed this code as in fastboot, `window.location.href` returns `undefined`. Therefore, will add the functionality later on in a separate PR.